ckanext/validation/plugin.py

```diff
@@ -171,8 +171,6 @@ def _process_schema_fields(self, data_dict):
 
     def before_create(self, context, data_dict):
         updated_data_dict = self._process_schema_fields(data_dict)
-        if updated_data_dict.get('schema'):
-            updated_data_dict['validation_status'] = 'unknown'
```

Comment:
Setting validation_status now happens using resource_patch from the resource_validation_run action.  It no longer needs to happen here.